### DISCUSSION TITLE
duda sobre comentarios de funcion

### DISCUSSION THREAD

**Discussion by damianperezz** [INGCOM-UNRN-P1/cursada-2023#93] [Preguntas y respuestas] [answered]
En la parte en la que se detallan las variables que entran en la funcion (@param), hay que mencionar las variables que se inicializan dentro de la funcion o solo las que son los argumentos?

**Comment by martinvilu** [comment, marked as answer]
Solo los argumentos de las funciones, dimos ejemplos en la última clase también.
